src/core/LadspaManager.cpp

```diff
@@ -122,14 +122,13 @@ LadspaManager::~LadspaManager()
 LadspaManagerDescription * LadspaManager::getDescription(
 						const ladspa_key_t & _plugin )
 {
-	if( m_ladspaManagerMap.contains( _plugin ) )
+	auto const it = m_ladspaManagerMap.find(_plugin);
+	if (it != m_ladspaManagerMap.end())
 	{
-		return( m_ladspaManagerMap[_plugin] );
-	}
-	else
-	{
-		return( nullptr );
+		return *it;
 	}
+
+	return nullptr;
```

Comment:
This can be condensed to `return it != m_ladspaManagerMap.end() ? *it : nullptr;`

Reply:
Done with commit a7335bb73d3.